test-app-reactive/src/main/kotlin/uk/gov/justice/digital/hmpps/testappreactive/config/WebClientConfiguration.kt

```diff
@@ -43,7 +43,6 @@ class WebClientConfiguration(
     reactiveUsernameAwareTokenRequestOAuth2AuthorizedClientManager(
       reactiveClientRegistrationRepository,
       reactiveOAuth2AuthorizedClientService,
-      timeout,
```

Comment:
this was confusing - we were using the same timeout for the client calls as well as the client credentials calls.  By providing a default in the library we don't need to specify the default here